modules/features/referrals/build.gradle.kts

```diff
@@ -0,0 +1,46 @@
+plugins {
+    alias(libs.plugins.android.library)
+    alias(libs.plugins.kotlin.android)
+    alias(libs.plugins.ksp)
+    alias(libs.plugins.hilt)
+    alias(libs.plugins.kotlin.parcelize)
+}
+
+android {
+    namespace = "au.com.shiftyjelly.pocketcasts.referrals"
+    buildFeatures {
+        buildConfig = true
+        compose = true
+    }
+}
+
+dependencies {
+    ksp(libs.dagger.hilt.compiler)
+    ksp(libs.hilt.compiler)
+
+    api(libs.dagger.hilt.android)
+    api(projects.modules.services.compose)
+    api(projects.modules.services.localization)
+    api(projects.modules.services.model)
+    api(projects.modules.services.repositories)
+    api(projects.modules.services.ui)
+
+    implementation(platform(libs.compose.bom))
+
+    implementation(libs.compose.material)
+    implementation(libs.compose.ui)
+    implementation(libs.compose.ui.tooling.preview)
+    implementation(libs.coroutines.reactive)
+    implementation(libs.hilt.navigation.compose)
+    implementation(libs.lifecycle.runtime.compose)
```

Comment:
There are some configuration issues here. If you run `./gradlew buildHealth` it will share advice with potential fixes.